### ISSUE TITLE
Bad presentation of untitled notes in overview screen

### ISSUE THREAD

**Issue by jech** [quillpad/quillpad#195]
When I don't give a title to a note, it appears in the overview screen with the title **Untitled**.  Duh.  I know it's untitled.

Please change the presentation of untitled notes so they appear with no title at all, thus saving precious vertical space.

**Comment by theAkito** [comment]
I personally prefer it having the word "Untitled", because it is displaying reality & reminds me to put a helpful title there.

I think, this should be an advanced option in the settings, so each user can decide, individually.

@jsixface I'd be willing to implement this as part of the Hacktoberfest event. Is that doable? 😄

**Comment by jsixface** [comment]
Sure. Appreciate it. 

I think having the word Untitled forces you to have a title. But it's just notes and agree that it doesn't need a title

**Comment by theAkito** [comment]
Well, I don't want this to be the default. I think, putting "Untitled" into an empty title makes sense, because it makes you put a relevant title there. So, if you worry about wasted space, just put a relevant title there & the space isn't wasted, anymore. Simple.

Here my [full explanation](https://github.com/quillpad/quillpad/pull/203#issuecomment-1782670873).

I would implement this as an option, togglable in the setting, but if that's not going to happen, I'm not interested in helping make this app's UX worse, by changing the default of this behaviour to a worse one.

**Comment by jech** [comment]
> I think, putting "Untitled" into an empty title makes sense, because it makes you put a relevant title there.

I most respectfully disagree.  I'm an adult human being, the app has no business telling me what I need to do.

> Here my full explanation.

Too many setting are not good, they make the app uselessly complicated.

**Comment by theAkito** [comment]
> I most respectfully disagree. I'm an adult human being, the app has no business telling me what I need to do.

I suggest you read up on how the human brain works & what psychology is.

It's not about the app forcing you to do anything. It's about being reminded of something, that you otherwise would not think about. It's a helpful method.

Additionally, you should re-consider your note-taking method, if none of your notes have titles. If you have 20 or more notes at some point, it's tough to distinguish which note has which purpose.
Notes with no titles ever are extremely inefficient for your brain.

> Too many setting are not good, they make the app uselessly complicated.

This statement is false.

It's not "useless", neither is it "complicated" to have many settings, which are well described.

& if you are an "adult human being" you should be perfectly able to handle more than 10 settings on a single page.

Plus, less options, means that the app is telling you what to do, because you cannot change the behaviour of what it's doing. 😆

**Comment by jsixface** [comment]
> I suggest you read up on how the human brain works & what psychology is.
> if you are an "adult human being" 

There is no call for being rude

**Comment by theAkito** [comment]
> There is no call for being rude

> I'm an adult human being, the app has no business telling me what I need to do.

Precisely. I don't see the point of this whole issue anymore. It's not an issue. It might be an option, for personal preference, at best. Otherwise, it's absolutely pointless. This is not about the app, this is about the personal problems of the guy who posted this.

**Comment by Tintedfireglass** [comment]
we could just keep it ass a toggle in settings if that is possible?

**Comment by opk12** [comment]
Just a heads up that #303 asks to use the first line of the note as the title.

**Comment by opk12** [comment]
The issue is even worse now that the cursor defaults to the body (#274), so you have to do an extra tap for each note just to avoid the ugly and useless `Untitled`.
